search/geocoder.hpp

```diff
@@ -262,6 +262,12 @@ class Geocoder
   WARN_UNUSED_RESULT bool GetTypeInGeocoding(BaseContext const & ctx, uint32_t featureId,
                                              Model::Type & type);
 
+  // Reorders maps in a way that prefix consists of maps intersecting with pivot, map with user location
+  // and maps with cities matched with query. Suffix consists of all other maps ordered by minimum
```

Comment:
nit: "matched to" to avoid double "with"

Reply:
done